Microsoft.Toolkit.Uwp.UI/Extensions/StringExtensions.cs

```diff
@@ -201,5 +181,17 @@ public static Quaternion ToQuaternion(this string text)
 
             static Quaternion Throw(string text) => throw new FormatException($"Cannot convert \"{text}\" to {nameof(Quaternion)}. Use the format \"float, float, float, float\"");
         }
+
+        private static string Unbracket(string text)
```

Comment:
Nit: please add full XML docs in here too 🙂

Reply:
Done 👍